public/app/plugins/datasource/tempo/SearchTraceQLEditor/utils.ts

```diff
@@ -81,9 +81,16 @@ const adHocValueHelper = (f: AdHocVariableFilter) => {
   if (intrinsics.find((t) => t === f.key)) {
     return f.value;
   }
+  if (parseInt(f.value, 10).toString() === f.value) {
+    return f.value;
+  }
   return `"${f.value}"`;
 };
 
+export const getTagWithoutScope = (tag: string) => {
+  return tag.replace(/^(event|link|resource|span)\./, '');
```

Comment:
Is `event` and `link` a Tempo 2 scope? 🤔 [docs](https://grafana.com/docs/tempo/latest/api_docs/#search-tags-v2) mention only resource, span and intrinsic. Could we use `this.languageProvider.tagsV2` here?

Reply:
Both of them are new scopes in recent (last month or so) versions of Tempo docker images, mostly just spoke about them in the Tempo monthly meeting.

This method `getTagWithoutScope` is actually only used with `this.languageProvider.getOptionsV1`.